inference/generativeai/llm-workshop/lab6-stream-with-pagination/stream_pagination_lmi.ipynb

```diff
@@ -577,8 +577,8 @@
     "            \"Action\": [\"sagemaker:ListEndpoints\", \"sagemaker:InvokeEndpoint\"],\n",
     "            \"Resource\": [\"arn:aws:sagemaker:*:*:endpoint/lmi*\"],\n",
     "            \"Effect\": \"Allow\",\n",
```

Comment:
Not sure if needed, but I removed the trailing comma here too and the other 2 similar ones before it

Reply:
574 and 569 here